developers/weaviate/client-libraries/typescript/v2_v3_migration.md

```diff
@@ -8,62 +8,34 @@ image: og/docs/client-libraries.jpg
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import FilteredTextBlock from '@site/src/components/Documentation/FilteredTextBlock';
-import PythonCode from '!!raw-loader!/_includes/code/client-libraries/python_v4.py';
 
 
 :::note Typescript client version
 The current Typescript client version is `v||site.typescript_client_version||`
 :::
 
-The `v3` Weaviate typescript client API is very different to the `v2` API. This guide will help you understand the major changes and how to migrate your code at a high level.
+The `v3` Weaviate Typescript client API is very different to the `v2` API. This guide will help you understand the major changes and how to migrate your code at a high level. 
+To start, `v3` refers to the client with the package name `weaviate-client` and `v2` refers to the package name `weaviate-ts-client`. 
+
+The `v3` client is meant for server development and will not work for web client development. If you need to develop a web client application, please use the `v2` client or use the `v3` [web client](#).
```

Comment:
Consider:
The `v3` client supports Node.js server based development. It does not support web client development.

To develop a web client based application, use the [`v2` client](#NEEDS_LINK) or the `v3` [web client](#NEEDS_LINK) when it is available.